### PR TITLE
doc: add early-access page

### PR DESCRIPTION
**Pull request by gctucker**
Add a page about the Early Access phase to make it easier for new users to get started.

### CODE REVIEW

**Review by nuclearcat — APPROVED** [review]
LGTM

**Comment by gctucker** [comment]
Deployed on staging:
https://static.staging.kernelci.org/docs/api/early-access/

File: doc/early-access.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+```
+
+> **Tip**: If you're wondering how it managed to find the API, take a look at
+> `/etc/kernelci/core/api-configs.yaml` from within the Docker container.
+
+* Once you've received your confirmation email with your randomly-generated
+  password, you should change it to use your own arbitrary one instead:
+
+```sh
+$ kernelci@d7e27fe0ba0b:~$ kci user change_password --user <your-user-name>
```
Comment (pawiecz) [inline doc/early-access.md:71]: minor: should the host prompt (`^$ `) be kept with the docker one?

I don't have strong preference for or against it - I like both differentiation from the first listing as well as the below ones being clean and concise

**Review by pawiecz — APPROVED** [review]
:+1:

File: doc/early-access.md (hunk 1 above)
Comment (gctucker) [reply doc/early-access.md:71]: Good point.  I think it's important to make it clear what is run directly on the host and what is in the container.  Let's see if I can clarify that, maybe like I did in the AKS README with first a step to start the container than assume the rest of the commands are all run from it, with some explanation.

Comment (gctucker) [reply doc/early-access.md:75]: OK I've added a small paragraph about it, please let me know if this makes sense.

Comment (gctucker) [reply doc/early-access.md]: (also removed a long container prompt as a result)

**Comment by gctucker** [comment]
I've now added a part about AzureFiles as that's also important, as a default way for users to store data alongside the API during the Early Access phase.

**Pull request merged**